GateWatcher/aioniq/tests/suricata_alert_event.json

```
@@ -0,0 +1,68 @@
+  },
+  "expected": {
+    "gatewatcher": {
+      "gcenter": [
+        "gcenter-sekoia.gatewatcher.com",
+        "gcenter-sekoia.gatewatcher.com"
+      ],
+      "gcap": "gcap-sekoia.gatewatcher.com",
+      "severity": 2,
+      "timestamp_analyzed": "2022-06-03T14:59:41.373Z",
+      "timestamp_detected": "2022-06-03T14:59:08.780Z",
+      "type": "suricata"
+    },
+    "http": {
+      "request": {
+        "method": "GET"
+      },
+      "response": {
+        "status_code": 200
+      }
+    },
+    "network": {
+      "transport": "TCP"
+    },
+    "observer": {
+      "hostname": "network.internal",
+      "name": "gcap-sekoia.gatewatcher.com",
+      "version": "0.2"
+    },
+    "related": {
+      "hosts": [
+        "network.internal"
+      ],
+      "ip": [
+        "9.8.7.6"
+      ]
+    },
+    "rule": {
+      "category": "Potesekoiaally Bad Traffic",
```

```suggestion
      "category": "Potentially Bad Traffic",
```

You have a strange typo here

---

Fixed. Thanks